build_library/build_image_util.sh

```diff
@@ -381,5 +386,8 @@ finish_image() {
             --target="${target}" --disk_image="${disk_img}" --noverity
       fi
     done
+    ${BUILD_LIBRARY_DIR}/generate_grub_hashes.py ${disk_img} /usr/lib/grub/ ${pcr_dir} ${COREOS_VERSION}
   fi
+  cd ${BUILD_DIR}
```

Comment:
Hm, zip doesn't have an equivalent to tar's -C? I don't know if working directory is or ever will be significant to other parts of the script but probably should use pushd/popd or similar scheme just to avoid confusing anyone in the future.